core/src/main/java/edu/ucr/cs/riple/core/metadata/index/Index.java

```diff
@@ -65,15 +66,15 @@ public enum Type {
   }
 
   /**
-   * Creates an instance of Index.
+   * Creates an instance of Index. Contents are accumulated from multiple sources.
    *
-   * @param path Path to the file to load the data from.
+   * @param paths ImmutableSet of paths to load the data from.
```

Comment:
Can u add some description around, what is the format expected for this file? maybe a sample row?

Reply:
Sure, actually there is no expected header. These classes can index any type of `.tsv` file where they contain information for creating instances of `Enclosed`. Yes I do agree that javadoc was not clear enough, please find the updated version c93236ba89e4e746392620f1011b0a23327105d9 and let me know if I should add anything else. Thanks.